azurelinuxagent/common/utils/cryptutil.py

```diff
@@ -128,3 +141,17 @@ def bits_to_bytes(self, bits):
                 index = 7
         return bytes(byte_array)
 
+    def decodeBase64ToFile(self, cacheFile, base64str):
```

Comment:
Python uses snake_case, please be consistent with that style.

Reply:
NIT: I don't like this method.  You should be able to compose it with existing base64 and file write methods instead of introducing a particular method.

Reply:
Will correct that as I see it anywhere else.